src/autocomplete/index.ts

```diff
@@ -7,11 +7,13 @@ class Autocomplete {
 
   state: Autocomplete.State = <any>{
     onHover: (event: MouseEvent) => {
+      const updateQuery = this.config.autocomplete.hoverQuery;
       const targets = this.activationTargets();
       if (this.isActive()) {
-        this.setActivation(targets, this.state.selected, false);
+        this.setActivation(targets, this.state.selected, false, updateQuery);
       }
-      this.setActivation(targets, Array.from(targets).findIndex((element) => element === event.target), true);
+      this.setActivation(targets, Array.from(targets).findIndex((element) => element === event.target),
```

Comment:
What is happening with this indentation?

Reply:
Just to avoid a long single line, if tslint ignore next line is preferred then I can do that.